include/sound/sof/topology.h

```diff
@@ -53,9 +53,10 @@ struct sof_ipc_comp {
 	uint32_t id;
 	enum sof_comp_type type;
 	uint32_t pipeline_id;
+	uint32_t core;
```

Comment:
I'm wondering whether we shouldn't use data fields more sparingly. We've got only one 32-bit reserved field left. And I'm pretty sure we won't need more than 255 cores any time soon... Other fields could be compressed too, but that would require a new major ABI version. Maybe when we do switch to a new major ABI version, we perform some such compression?

Reply:
@lyakh Agree. We need to figure it out with the next major bump.